style/desktop.css

```diff
@@ -240,12 +241,330 @@
     /* width: 400px; */
 
     position: relative;
-    background-image: url("/src/assets/imgs/bg-mobile-see.png");
     background-repeat: no-repeat;
     background-size: cover;
     background-position: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: end;
+    align-items: start;
+    gap: 2rem;
+    padding: 3rem 0 1rem 0;
+  }
+
+  .see-project-1 {
+    width: 100%;
+    background-image: url("/src/assets/imgs/Img Placeholder1.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .see-project-2 {
+    width: 100%;
+    background-image: url("/src/assets/imgs/Img Placeholder2.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .see-project-3 {
+    width: 100%;
+    background-image: url("/src/assets/imgs/Img Placeholder6.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .see-project-4 {
+    width: 100%;
+    background-image: url("/src/assets/imgs/Img Placeholderzzzz.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .see-project-5 {
+    width: 100%;
+    background-image: url("/src/assets/imgs/Img Placeholder2.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .see-project-6 {
+    width: 100%;
+    background-image: url("/src/assets/imgs/Img Placeholder6.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .Fourth-head {
+    margin: 0;
+    width: 20rem;
+    text-align: left;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    color: #fff;
+    line-height: 44px;
+    padding: 5rem 0 0.5rem 1rem;
+  }
+
+  .project-detail {
+    max-width: 25rem;
+    width: 100%;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    color: #fff;
+    line-height: 24px;
+    padding-left: 1rem;
+  }
+
+  .my-project {
+    padding-left: 1rem;
+    margin-top: 1.2rem;
+  }
+
+  .my-project a {
+    color: #fff;
+    background: #ffffff3d;
+  }
+
+  .card-btn {
+    width: 488px;
+    height: 4rem;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+  }
+
+  .hidden {
+    display: none;
+  }
+
+  /* style about section */
+
+  .about-section {
+    background-color: #1c1a19;
+    justify-content: start;
+    align-items: center;
+    position: relative;
+    margin-top: 5rem;
+    padding: 5rem 8rem 0.6rem 8rem;
+  }
```

Comment:
As mentioned above, I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

>### The Figma design
>
><img width="644" alt="Screenshot 2023-05-04 at 5 27 04 AM" src="https://user-images.githubusercontent.com/91237525/236165670-62a8ce29-0053-44e0-b055-68d34599a2e6.png">
>
>### Your design
>
><img width="1435" alt="Screenshot 2023-05-04 at 5 28 34 AM" src="https://user-images.githubusercontent.com/91237525/236166092-4b076866-4d43-4b14-924a-3cd9279b7e9e.png">

Reply:
## what i did
* fixed the size of illustration
>
![about](https://user-images.githubusercontent.com/99332080/236203671-34e8cc3c-8322-4c10-a85f-0a07ebef3ac1.png)

Reply:
Fixed.